terraform/main.tf

```diff
@@ -9,6 +9,12 @@ terraform {
   required_providers {
     azurerm = {
       source = "hashicorp/azurerm"
+      # Make sure this matches the version in ../nix/devshell.nix.
+      # This will not pin the version (it's pinned in devshell.nix),
+      # but makes terraform emit an error message in case the azurerm
+      # version cached in local .terraform directory for one reason
+      # or antoher does not match what is stated below.
```

Comment:
s/antoher/another

Reply:
Thanks, fixed in https://github.com/tiiuae/ghaf-infra/commit/b6abe2966b1e2871232b31fcc2c2c9690582b235